cfme/containers/provider/__init__.py

```diff
@@ -5,6 +5,12 @@
 
 import fauxfactory
 from navmazing import NavigateToSibling, NavigateToAttribute
+from cfme.base.login import BaseLoggedInPage
+from widgetastic_patternfly import SelectorDropdown, Button, Dropdown
+from widgetastic.widget import Text
+import random
+from utils.wait import wait_for
```

Comment:
Can you, please, order the imports here?

First come python std library imports, then other libraries, then local project imports.
Each of these groups first `import ...` and after that `from ... import ...`, and in those the names ordered ascending :)